src/main.py

```diff
@@ -34,130 +34,25 @@
 
 
 def load_rawdata_into_pkl():
-    """with open(PATH_TO_DATASET, 'rb') as f:
-    dataset = pickle.load(f)
-    AS = AlignmentSearch(dataset=dataset)
-    prompt = "What would be an idea to solve the Alignment Problem? Name the Lesswrong post by Quintin Pope that discusses this idea."
-    answer = AS.search_and_answer(prompt, 3, HyDE=False)
-    print(answer)
-    """  
-    # List of possible sources:
-    all_sources = ["https://aipulse.org", "ebook", "https://qualiacomputing.com", "alignment forum", "lesswrong", "manual", "arxiv", "https://deepmindsafetyresearch.medium.com", "waitbutwhy.com", "GitHub", "https://aiimpacts.org", "arbital.com", "carado.moe", "nonarxiv_papers", "https://vkrakovna.wordpress.com", "https://jsteinhardt.wordpress.com", "audio-transcripts", "https://intelligence.org", "youtube", "reports", "https://aisafety.camp", "curriculum", "https://www.yudkowsky.net", "distill", 
-                   "Cold Takes", "printouts", "gwern.net", "generative.ink", "greaterwrong.com"] # These last do not have a source field in the .jsonl file
-
-    # List of sources we are using for the test run:
-    custom_sources = [
-        "https://aipulse.org", 
-        "ebook", 
-        "https://qualiacomputing.com", 
-        "alignment forum", 
-        "lesswrong", 
-        "manual", 
-        "arxiv", 
-        "https://deepmindsafetyresearch.medium.com/", 
-        "waitbutwhy.com", 
-        "GitHub", 
-        "https://aiimpacts.org", 
-        "arbital.com", 
-        "carado.moe", 
-        "nonarxiv_papers", 
-        "https://vkrakovna.wordpress.com", 
-        "https://jsteinhardt.wordpress.com", 
-        "audio-transcripts", 
-        "https://intelligence.org", 
-        "youtube", 
-        "reports", 
-        "https://aisafety.camp", 
-        "curriculum", 
-        "https://www.yudkowsky.net", 
-        "distill",
-        "Cold Takes",
-        "printouts",
-        "gwern.net",
-        "generative.ink",
-        "greaterwrong.com"
-    ]
-    
-    dataset = create_dataset.Dataset(
-        custom_sources=custom_sources, 
-        rate_limit_per_minute=3500, 
-        min_tokens_per_block=200, max_tokens_per_block=300, 
-        # fraction_of_articles_to_use=1/150,
+    dataset = create_dataset.ChunkedARD(
+        min_tokens_per_block=200, max_tokens_per_block=300
     )
     dataset.get_alignment_texts()
-    
-    print(len(dataset.embedding_strings))
-    print(dataset.total_word_count)
-    print(dataset.total_block_count)
-    print(dataset.articles_count)
-    
     dataset.get_embeddings()
     dataset.save_data()
-    
+    dataset.show_stats()
+
+def load_pkl_and_display_stuff():
+    with open(PATH_TO_DATASET_DICT_PKL, 'rb') as f:
+        dataset_dict = pickle.load(f)    
+    print(dataset_dict)
+
 @retry(wait=wait_random_exponential(min=1, max=20), stop=stop_after_attempt(4))
 def get_embedding(text: str) -> np.ndarray:
```

Comment:
is this function used?

Reply:
The get_embedding ones isn't currently, it's just there to be used to test whether the openAI embeddings worked correctly. I can check whether dataset.embeddings[0] == get_embedding(dataset.metadata[0])